doc/crypt.tex

```diff
@@ -6710,12 +6710,12 @@ \subsection{URL--safe 'base64url' encoding}
 \begin{verbatim}
 int base16_encode(const unsigned char *in,  unsigned long  inlen,
                                  char *out, unsigned long *outlen,
-                                 int  caps);
+                        unsigned long  options);
```

Comment:
I would prefer `unsigned int` which is 32bit pretty everywhere. With `unsigned long` there might be a trouble that it is only 32bit on Windows/x86 but 64bit on many other platforms.